.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 1. AIRAC (2507) - Updated Newcastle (EGNT) SMR Runway Markings - thanks to @clc0609 (Coby Chapman)
 2. Enhancement - Created Mildenhall (EGUN) SMR - thanks to @clc0609 (Coby Chapman)
 3. AIRAC (2507) - Removed danger area EGD206 - thanks to @clc0609 (Coby Chapman)
+4. Bug - Fix Yeovilton (EGDY) Runway Designator - thanks to @lucutz1234
```

Comment:
```suggestion
4. Bug - Fixed Yeovilton (EGDY) Runway Designator - thanks to @lucutz1234
```
Per the [style guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#entry-summary-), changelog entries should be past tense.